docs/en/basic/Command-Line-Interface.md

```diff
@@ -187,6 +203,11 @@ The usage of this command includes:
 -Dalluxio.underfs.s3.endpoint=<endpoint_url> -Dalluxio.underfs.s3.disable.dns.buckets=true
 ```
 
+### readJournal
```

Comment:
`readJournal` is a developer facing command that will be better to put in the `contributor-tool.md` and I updated this docs in https://github.com/Alluxio/alluxio/pull/9249/files according to @ggezer 's newest journalTool changes

Reply:
removed